coverage.md

```diff
@@ -16,6 +16,7 @@ Manual Coverage
 | As a user, I can sync and skip specific type (srpm) | NO |  |
 | As a user, I can sync opensuse repository | NO |  |
 | As a user, I can sync from a mirror list | YES |  |
+| As a user, I can sync from a mirror list with comments | YES |  |
```

Comment:
Coverage is about tests. I can see how it is not clear from this page. Apologies. I'll update the language here.

We aim to add a test with every fix or feature.
I believe the link you shared in the issue, the mirror list for epel8, would be a good addition to our daily tests.
We have functional tests which run with every PR and we have performance/daily tests which run once a day.
 
Could you add one similar to this https://github.com/pulp/pulp_rpm/blob/3.6/pulp_rpm/tests/performance/test_sync.py#L167-L169 ? This way we'll keep this line you added to the coverage and we'll test daily that a mirror list works.